cms/CONTENTFUL.md

```diff
@@ -1,5 +1,29 @@
 # Contentful
 
+## Terminology
+
+- `CMS`: Content Management System.
+- `Space`: The Early Years Recovery content, team management, release schedule, mock data for application test, all exist within a space.
+- `Alias`: Content in environments is accessed using aliases. The target an alias points to can be changed. We have access to three aliases.
+- `Environment`: Content exists within an environment. We have access to four environments.
+- `Content model`: The conceptual entities used to author content. Used to define attributes, validations and control the editor interface.
```

Comment:
Wonder if this definition of `content model` is a bit abstract - would it be helpful to include an example here?